zookeeper-server/src/main/java/org/apache/zookeeper/server/PrepRequestProcessor.java

```diff
@@ -902,6 +909,7 @@ protected void pRequest(Request request) throws RequestProcessorException {
             }
         }
         request.zxid = zks.getZxid();
+        ServerMetrics.PREP_PROCESSOR_QUEUE_TIME.add(Time.currentElapsedTime() - request.prepQueueStartTime);
```

Comment:
What happens if prepQueueStartTime stay the default value -1 ?

Reply:
A request that has got here must have gone through processRequest(), which records prepQueueStartTime(). Can't think of a scenario that it stays -1.